.vale/styles/config/vocabularies/book/accept.txt

```diff
@@ -1,3 +1,12 @@
 HAL
 esp-hal
 RTIC
+IEEE
+BLE
+USB
+WROOM
+(?i)NOW
```

Comment:
Is this syntax explained somewhere?

Reply:
These are flags as documented by the `regex` crate:  
https://docs.rs/regex/latest/regex/#grouping-and-flags